src/data.toml

```diff
@@ -1,18 +1,22 @@
 ["1196075698301968455"] # pyoncord, responses for triggered intents
 undetailed_question = "The information you provided is not detailed enough to answer your question.\nPlease provide more information or rephrase your question."
-get_addons = "Bunny is based on Vendetta, so you may install any of its plugins:\n\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n-# You are free to request plugin urls to be added to this list\n\n- You may install any [Vendetta](<https://discord.gg/rJgnCJ3Kzm>)/[Enmity](<https://discord.gg/rMdzhWUaGT>)/[Revenge](<https://discord.com/invite/ddcQf3s2Uq>) theme you like, do note that old themes may be incompatible\n\nYou can find a curated list of broken plugins at https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
-installation_instructions = "### Android\n\n- **Root** with LSPosed - [BunnyXposed](<https://github.com/pyoncord/BunnyXposed/releases/latest>)\n- **Non-root** - [BunnyManager](<https://github.com/pyoncord/BunnyManager/releases/latest>)\n### iOS\n\n- **Jailed**\n - Pre-patched `.ipa` - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n - AltSource Repo - [Pyoncord App Repo](<https://raw.githubusercontent.com/pyoncord/BunnyTweak/main/app-repo.json>)\n-# Note: TrollStore may display an encryption warning which you can ignore\n- **Jailbroken**\n - Tweak Debian Package - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n\nKeep in mind that this channel is not sideload-, jailbreak nor root support.\nPlease keep questions pertaining to those topics elsewhere."
+get_addons = "Bunny offers compatibility with Vendetta & Revenge plugins and themes:\n\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\nYou are free to request plugin urls to be added to this list\n\n- You may install any  [Vendetta](<https://discord.gg/rJgnCJ3Kzm>)/[Enmity](<https://discord.gg/enmity>)/[Revenge](<https://discord.com/invite/ddcQf3s2Uq>) themes you like, do note that old themes may be incompatible\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
```

Comment:
there's an extra whitespace that snuck in there and `at` was more appropriate flow wise

Reply:
https://github.com/pyoncord/autosupport/pull/1/files/4a900b44db70a26becde935f48c7700a6fcad484#r1691561194
Please point where it is.